app/api/helpers/utilities.py

```diff
@@ -1,7 +1,27 @@
 # PLEASE PUT ALL FUNCTIONS WHICH PERFORM GENERAL FORMATTING ON ANY DATATYPE WITHOUT USING ANY
 # MODULES RELATED TO THE EVENT-SYSTEM i.e FUNCTIONS RELATED TO DB MODELS, OTHER API MODULES
 # OR OTHER HELPER MODULES
+from dateutil import parser
+from pytz import timezone
 
 
 def dasherize(text):
     return text.replace('_', '-')
+
+
+def changeToTimezone(result):
+    tz = timezone(result['data']['attributes']['timezone'])
```

Comment:
You are expecting single object while you will receive array of objects in result['data'] so basically your tz is None
I tried a simple fix. Please check this :
```
def changeToTimezone(result):
    new_data = []
    for data in result['data']:
        tz = timezone(data['attributes']['timezone'])

        start_date_datetime = tz.localize(parser.parse(data['attributes']['starts-at']).replace(tzinfo=None))
        data['attributes']['starts-at'] = start_date_datetime.isoformat()

        end_date_datetime = tz.localize(parser.parse(data['attributes']['ends-at']).replace(tzinfo=None))
        data['attributes']['ends-at'] = end_date_datetime.isoformat()

        if data['attributes']['schedule-published-on']:
            schedule_pub_datetime = tz.localize(
                parser.parse(data['attributes']['schedule-published-on']).replace(tzinfo=None))
            data['attributes']['schedule-published-on'] = schedule_pub_datetime.isoformat()
        new_data.append(data)
    result['data'] = new_data
    return result
```